### PR TITLE
Replaced YML based config with SQLite based models.

### PR DESCRIPTION
**Pull request by saleem-latif**
This PR removes all usages of `yml` based config file with new database backed models and tables. Testing will include everything. including login/logout flow and all aspects of repo sync/unsync process.

### CODE REVIEW

**Review by Basit-qc — APPROVED** [review]
Great work @saleem-latif and great job on adding unit tests 👏 Had a quick look and things look fine.

File: src/main/java/org/intellij/sdk/codesync/database/models/Model.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package org.intellij.sdk.codesync.database.models;
+
+public class Model {
```
[the remark this answers is not in the document]
Comment (saleem-latif) [reply src/main/java/org/intellij/sdk/codesync/database/models/Model.java:3]: I created this base class to host common methods that are used by all models.

File: src/main/java/org/intellij/sdk/codesync/utils/Queries.java [hunk 2]
```diff
@@ -1,50 +0,0 @@
-package org.intellij.sdk.codesync.utils;
-
-public class Queries {
-
-    public static class User{
-
-        public static final String TABLE_EXIST = Queries.table_exist("user");
-
-        //TODO Turn this into multiline string.
-        public static final String CREATE_TABLE = "CREATE TABLE IF NOT EXISTS user(EMAIL TEXT PRIMARY KEY, ACCESS_TOKEN TEXT, SECRET_KEY TEXT, ACCESS_KEY TEXT, IS_ACTIVE INT)";
```
[the remark this answers is not in the document]
Comment (saleem-latif) [reply src/main/java/org/intellij/sdk/codesync/utils/Queries.java:10]: Yeah, Sorry. I think I was also mistaken.

**Comment by saleem-latif** [comment]
@Basit-qc I think we can ignore these vulnerability reports as those are for server side code however we are running database on the client side and not need to secure it.

File: src/main/java/org/intellij/sdk/codesync/database/queries/MigrationsQueries.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+public class MigrationsQueries extends CommonQueries {
+    String tableName;
+
+    public MigrationsQueries(String tableName) {
+        this.tableName = tableName;
+    }
+
+    public String getCreateTableQuery() {
+        return String.format("CREATE TABLE IF NOT EXISTS %s (", this.tableName) +
```
[the remark this answers is not in the document]
Comment (saleem-latif) [reply src/main/java/org/intellij/sdk/codesync/database/queries/MigrationsQueries.java:11]: Good suggestion, I will add it to documentation after this one is well tested and deployed.

**Pull request merged**